Bottom: 3bf1f85f839e36f2e139275bee8e8f5e0a7ad4d1
Top:    e95ca5b38bc9fa67b1cb6d454b1b8f020bf6a8e2
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-09-19 11:16:26 +0100

hid/gtk: Fix Pan action to stop after button release outside window

Previously, we required in-view coordinates to stop and start the
pan-action, and when the button was released outside the viewport,
the code was querying the user to click at a particular loacation.

Since the pan action doesn't even use the passed in coordinates,
simply remove the requirement for the x-y coordinates from this
action.

Closes-bug: lp-699307


---

diff --git a/src/hid/gtk/gtkhid-main.c b/src/hid/gtk/gtkhid-main.c
index 9b239e1..a83ee2e 100644
--- a/src/hid/gtk/gtkhid-main.c
+++ b/src/hid/gtk/gtkhid-main.c
@@ -2030,7 +2030,7 @@ HID_Action ghid_main_action_list[] = {
   {"LayerGroupsChanged", 0, LayerGroupsChanged},
   {"LibraryChanged", 0, LibraryChanged},
   {"Load", 0, Load},
-  {"Pan", N_("Click on a place to pan"), PanAction, pan_help, pan_syntax},
+  {"Pan", 0, PanAction, pan_help, pan_syntax},
   {"PCBChanged", 0, PCBChanged},
   {"PointCursor", 0, PointCursor},
   {"Popup", 0, Popup, popup_help, popup_syntax},
